plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java

```diff
@@ -69,6 +72,22 @@ public void setUp()
         assertQuery("SELECT count(*) FROM base_table2", "VALUES 3");
     }
 
+    @Test
+    public void testShowTables()
+    {
+        assertUpdate("CREATE MATERIALIZED VIEW materialized_view_show_tables_test AS SELECT * FROM base_table1");
+        SchemaTableName storageTableName = getStorageTable(
+                new QualifiedObjectName("iceberg", "tpch", "materialized_view_show_tables_test"));
+
+        Set<String> expectedTables = ImmutableSet.of("base_table1", "base_table2", "materialized_view_show_tables_test", storageTableName.getTableName());
+        Set<String> actualTables = computeActual("SHOW TABLES").getOnlyColumnAsSet().stream()
+                .map(String.class::cast)
+                .collect(toImmutableSet());
+        assertThat(actualTables).containsAll(expectedTables);
```

Comment:
why not equals?

Reply:
it's to avoid concurrency issues as the test is not single threaded